hieradata/common.yaml

```diff
@@ -293,19 +293,19 @@ profile::buildmaster::agents_setup:
     tempDir: '/tmp'
 profile::buildmaster::agent_images:
   ec2_amis:
-    ubuntu-amd64: ami-0794388cd14c63313
-    windows-amd64: ami-0c150f883f409b9c7
-    ubuntu-arm64: ami-0bff468cafbf20c0e
+    ubuntu-amd64: None
```

Comment:
err???

Reply:
It sounds like that the updatecli is not correct: the script should fail instead of printing "None". Let me fix that (while adding a weekly timer to this updatecli process)

Reply:
This PR will auto-update once the AMI associated top the tag 0.5.0 will have been built successfully